src/DevTools/setupTestWrapper.tsx

```diff
@@ -110,7 +132,14 @@ export const setupTestWrapperTL = <T extends OperationType>({
       return MockPayloadGenerator.generate(operation, mockResolvers)
     })
 
-    return { ...view, env }
+    return {
+      ...view,
+      env,
+      mockResolveMostRecentOperation: mockResolveMostRecentOperationWithEnv(
+        env
+      ),
+      mockRejectMostRecentOperation: mockRejectMostRecentOperationWithEnv(env),
```

Comment:
[blocking]: Sorry if this wasn't clear, but we should be copying Eigen's approach exactly, to a T, minus the few additions we discussed in the final comments. We don't need new files, or new `apply` functions, or new types, or even comments, (other than a basic usecase example). and so on. 

It also might be helpful to close this PR and open up a fresh one.